.orchestra/config/components/expat.yml

```diff
@@ -0,0 +1,23 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+#@ load("/lib/cmake.lib.yml", "typical_cmake_builds")
+
+---
+#@ def build_args():
+test: false
+source_url: "https://github.com/libexpat/libexpat/releases/download/R_2_4_8/expat-2.4.8.tar.gz"
+use_asan: false
+build_dependencies:
+  - cmake
+  - glibc
```

Comment:
I think `glibc` is implied by `host-c-toolchain`, you can drop it. One day we might switch to musl `:)`.